File: src/main/java/com/EmosewaPixel/pixellib/recipes/SimpleRecipeList.java
package com.EmosewaPixel.pixellib.recipes;

import net.minecraft.util.ResourceLocation;

//This is a Recipe List for Simple Machine Recipes
public class SimpleRecipeList extends AbstractRecipeList<SimpleMachineRecipe, SimpleRecipeBuilder> {
    public SimpleRecipeList(ResourceLocation name, int maxInputs, int maxOutputs) {
        super(name, maxInputs, maxOutputs);
    }

    @Override
    public SimpleRecipeBuilder recipeBuilder() {
        return new SimpleRecipeBuilder(this);
    }
}

File: src/main/java/com/EmosewaPixel/pixellib/recipes/SimpleRecipeList.kt
package com.EmosewaPixel.pixellib.recipes

import net.minecraft.util.ResourceLocation

//This is a Recipe List for Simple Machine Recipes
class SimpleRecipeList(name: ResourceLocation, maxInputs: Int, maxOutputs: Int) : AbstractRecipeList<SimpleMachineRecipe, SimpleRecipeBuilder>(name, maxInputs, maxOutputs) {
    override fun recipeBuilder() = SimpleRecipeBuilder(this)
}
